test/generic_adapter_test.rb

```diff
@@ -0,0 +1,43 @@
+class GenericAdapterTest < Minitest::Test
```

Comment:
Super lame tests to just confirm we can use the NullODBCAdapter to do postgres operations

If we do run into issues in production with the adapter, this might end up being the right place to add additional tests confirming that we've fixed any specific issues